scripts/buildSqlModule.js

```diff
@@ -0,0 +1,35 @@
+const path = require("path");
+const fs = require("fs");
+
+const sqlDir = path.join(__dirname, "../sql");
+const sqlFiles = fs
+  .readdirSync(sqlDir)
+  .filter((file) => /^[0-9]{6}\.sql$/.test(file))
+  .sort();
+
+/**
+ * Turns `str` into a String.raw expression, safely escaping backticks and
+ * placeholder strings.
+ */
+function escape(str) {
+  return (
+    "String.raw`" +
```

Comment:
Bit of a pain, sorry about this, but I want the file contents to start on the next line to make it easier to read:

```suggestion
    "String.raw`\\\n" +
```